scripts/upgradeCss.mjs

```diff
@@ -0,0 +1,138 @@
+// prereq
+// yarn add @kristoferbaxter/estree-walker acorn magic-string -D -W
+import { execSync } from 'child_process'
+import * as fs from 'fs'
+import prettier from 'prettier'
+import { walk } from '@kristoferbaxter/estree-walker'
+import * as acorn from 'acorn'
+import MagicString from 'magic-string'
+
+async function main() {
+  await refactorCssInJs()
+  runBuild()
+  copyBuiltCssToSrc()
+  adjustPackageJson()
+  copyPostcssConfig()
+  rmOldCssJs()
+  refactorRootExports()
+}
+
+async function refactorRootExports() {
+  const css = readFile('src/index.ts')
+
+  const ast = acorn.parse(css, ACORN_OPTS)
+  const magic = new MagicString(css)
+
+  walk(ast, {
+    enter(node, parent, prop, index) {
+      if (node.type === 'ExportNamedDeclaration') {
+        if (/css/.test(node.source.value)) {
+          const [start, end] = node.range
+          magic.remove(start, end)
+        }
+      }
+    }
+  })
+
+  writeFile('src/index.ts', magic.toString())
+}
+
+function rmOldCssJs() {
+  fs.rmSync('src/css/index.ts')
+}
+
+function copyPostcssConfig() {
+  fs.copyFileSync('../avatar/postcss.config.js', 'postcss.config.js')
+}
+
+function adjustPackageJson() {
+  const pkg = JSON.parse(readFile('package.json'))
+
+  pkg.scripts['build:css'] = 'postcss-compile'
+  pkg.style = 'dist/esm/css/index.css'
+  if (pkg.peerDependencies) {
+    const { glamor, ...rest } = pkg.peerDependencies
+    pkg.peerDependencies = rest
+  }
+  if (pkg.sideEffects.toString() === 'false') {
+    pkg.sideEffects = ['**/*.css']
```

Comment:
What does setting this mean for tree shaking?

Reply:
It means that the css file should be considered to provide a side effect. It can't be tree-shaken and keep the program in tact -- in this case, the visuals.